libs/ui/src/components/NeoSelect/NeoSelect.vue

```diff
@@ -1,36 +1,48 @@
 <template>
-  <o-field :label="label">
-    <o-select :placeholder="placeholder">
-      <option
-        v-for="(option, index) in options"
-        :key="index"
-        :value="option.value">
-        {{ option.text }}
-      </option>
-    </o-select>
-  </o-field>
+  <o-select
+    v-model="selected"
+    :placeholder="placeholder"
+    class="neo-select"
+    :class="{ 'is-expanded': expanded }"
```

Comment:
```suggestion
    :class="{ 'o-ctrl-sel--expanded': expanded }"
```

we should be able to do this without custom prop